stdlib/genericpath.pyi

```diff
@@ -36,6 +39,9 @@ def getsize(filename: FileDescriptorOrPath) -> int: ...
 def isfile(path: FileDescriptorOrPath) -> bool: ...
 def isdir(s: FileDescriptorOrPath) -> bool: ...
 
+if sys.version_info >= (3, 12):
+    def islink(path: FileDescriptorOrPath) -> bool: ...
```

Comment:
`os.lstat(path)` is annotated with `StrOrBytesPath`, not `FileDescriptorPath`. Why the difference?

Reply:
My bad, I looked at `stat()`, not `lstat()` 👍 
Good catch!